backend/migrations/20201205141817_teachers.js

```diff
@@ -1,34 +1,41 @@
-const { tableTeachers } = require('../src/models/teacher')
-
+const { tableTeachers } = require("../src/models/teacher");
 
 exports.up = (knex) => {
   return knex.schema.createTable(tableTeachers, (table) => {
-    table.increments('TeacherID').primary() 
-    table.text('PhoneNumber').notNullable()
-    table.text('Email')
-    table.text('FullName').notNullable()
-    table.unique(['PhoneNumber', 'FullName'])
-    table.text('Source')
-    table.integer('NativeLanguageID').references('NativeLanguageID').inTable('nativeLanguages').onDelete('CASCADE')
-    table.integer('SecondLanguageID').references('NativeLanguageID').inTable('nativeLanguages').onDelete('CASCADE')
-    table.enu('EnglishProficiency', [
+    table.increments("TeacherID").primary();
+    table.text("PhoneNumber").notNullable();
+    table.text("Email");
+    table.text("FullName").notNullable();
+    table.unique(["PhoneNumber", "FullName"]);
+    table.text("Source");
+    table
+      .integer("NativeLanguageID")
+      .references("NativeLanguageID")
+      .inTable("nativeLanguages")
+      .onDelete("CASCADE");
+    table
+      .integer("SecondLanguageID")
+      .references("NativeLanguageID")
+      .inTable("nativeLanguages")
+      .onDelete("CASCADE");
+    table.enu("EnglishProficiency", ["No", "Little", "Simple", "Intermediate"]);
+    table.enu("LanguageProficiency", [
       "No",
       "Little",
       "Simple",
-      "Intermediate"
-    ])
-    table.enu('LanguageProficiency', [
-        "No",
-        "Little",
-        "Simple",
-        "Intermediate"
-      ])
-    table.text('Notes')
-    table.integer('StatusID').references('StatusID').inTable('statuses').onDelete("CASCADE")
-    table.timestamps(true, true)
-  })
-}
+      "Intermediate",
+    ]);
+    table.enu("TeachingExperience", ["Yes", "No"]);
```

Comment:
This should be in a new migration file.

Reply:
can create new migration file with command in backend: `npm run db:migrate:make add_teaching_experience`
where `add_teaching_experience` is the filename. You can use another name if you like as well